### ISSUE TITLE
Pop-up Problems

### ISSUE THREAD

**Issue by Ninjakitten** [Kadauchi/prolific-assistant#5]
I greatly appreciate your quick updating of the extension, but there's a few problems I'm noticing...

![image](https://user-images.githubusercontent.com/5254533/42686499-ffb6adae-864a-11e8-8097-837b00354bc4.png)

1) As you can see, it thinks there are five studies for me at present.  But none of them show up on the site either by reloading the studies page or clicking the links, and though it's been over ten minutes, the notifications aren't going away. (ETA: okay, it's down to 4 now, so I'm guessing the markers remain because the studies are still open -- that makes sense. Unfortunately they're still studies I can't actually get to...)
2) The 'reward' is saying things like 50.00 and 375.00 instead of .50 and 3.75.
3) Neither 'time allowed' nor 'completion time' seem to be getting data.

Thank you for your work!

**Comment by Kadauchi** [comment]
I'm unable to reproduce this. Can you uninstall then reinstall the extension? 

After that more than likely doesn't fix anything, what version of Chrome are you using?

**Comment by Ninjakitten** [comment]
I didn't have a chance to uninstall/reinstall yet, but it seems to have (mostly?) fixed itself anyway! The one remaining issue is that it doesn't give a "time allowed". Otherwise everything seems perfect, now.  Thank you again for your work!

**Comment by victrixx** [comment]
I have the same issue since Prolific's site update.

The new plugin worked correctly maybe once.
Since then it shows notifications of survey not available to me.
I already tried reinstalling it without luck.

**Comment by Ninjakitten** [comment]
So, I restarted my computer and Chrome said it hadn't shut down properly, so I restored the tabs.  Now, I'm back to the original problem I reported, somehow.  I tried uninstalling and reinstalling PA, but nothing has changed.

I was using Chrome version 67.0.3396.99 (Official Build) (64-bit) on Win 10 Pro, and just updated to version 68.0.3440.75 (Official Build) (64-bit), with no change.  I wish I could figure out what might have made it fix itself last time...
